packages/opencensus-nodejs/README.md

```diff
@@ -85,12 +143,12 @@ OpenCensus collects distributed tracing. It is able to do so by propagating span
 
 ## Exporters
 
-OpenCensus can export trace data to various backends. Currently, OpenCensus supports:
+OpenCensus can export trace and stats data to various backends. Currently, OpenCensus supports:
 
-- [Stackdriver](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-stackdriver/README.md)
-- [Zipkin](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-zipkin/README.md)
-- [Z-Pages](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-zpages/README.md)
-- [Jaeger](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-jaeger/README.md)
+- [Stackdriver](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-stackdriver/README.md) for trace and stats
+- [Zipkin](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-zipkin/README.md) for trace
+- [Z-Pages](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-zpages/README.md) for trace
```

Comment:
This is also for trace and stats, right?

Reply:
Yep, since the user imports the same package for both trace and stats my idea is to have their usage in the same README. I added a _for trace and stats_ at the end to ensure what is supported by each one.